Bottom: 6f4be71c5cde5b650bd3c66ca4879d1a02851ff1
Top:    791212eccf3dbce475eda9edae566cd58458de2e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 18:56:00 +0000

Refresh of autoroute-c-remove-unnecessary

---

diff --git a/src/autoroute.c b/src/autoroute.c
index d7d6aa1..80aad25 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -1024,7 +1024,6 @@ CreateRouteData ()
    * this saves on searching the trees to find the nets
    */
   /* use the DRCFLAG to mark objects as they are entered */
-  ResetConnections (false);
   Nets = CollectSubnets (false);
   {
     routebox_t *last_net = NULL;
